test/Game.test.js

```diff
@@ -0,0 +1,18 @@
+import { GameBoard } from '../src/scripts/Game';
+
+describe('should check is a valide move', () => {
+  let gameLogic =  GameBoard([null, null, null, null, null, null, null, null, null]);
+  
+  it('should return the board array', () => {
+    expect(gameLogic.board).toStrictEqual([null, null, null, null, null, null, null, null, null]);
+  });
+  
+});
+
+describe('should check the move is added to board array', () => {
+  let gameLogic = GameBoard([null, null, null, null, null, null, null, null, null]);
+  gameLogic.game(5, gameLogic.board);
+  it('should add X to index 5', () => {
```

Comment:
- [ ] [OPTIONAL] It's always a good idea to test different test cases to make sure it's not just passing by accident.